mysql/check.py

```diff
@@ -851,10 +851,23 @@ def _get_replica_stats(self, db):
         try:
             with closing(db.cursor(pymysql.cursors.DictCursor)) as cursor:
                 replica_results = {}
+
                 cursor.execute("SHOW SLAVE STATUS;")
-                slave_results = cursor.fetchone()
-                if slave_results:
-                    replica_results.update(slave_results)
+                slave_results = cursor.fetchall()
+                if len(slave_results) > 0:
+                    for slave_result in slave_results:
+                        # MySQL <5.7 does not have Channel_Name
+                        if 'Channel_Name' in slave_result:
```

Comment:
I think this is a better way as well, it saves some space and looks a bit nicer.